src/mumble/ManualPlugin.cpp

```diff
@@ -24,6 +24,9 @@ static bool bActive = true;
 static int iAzimuth = 0;
 static int iElevation = 0;
 
+static const QString defaultContext = QString::fromUtf8("Mumble");
```

Comment:
```suggestion
static const QString defaultContext = QString::fromLatin1("Mumble");
```

Reply:
Why do you prefer `fromLatin1`? I always felt more comfortable using UTF instead of any other encoding as it's more universal :thinking:

Reply:
`fromUtf8()` is preferred when the string may be Unicode, in our case we're sure it's not.

Reply:
But isn't Unicode identical to ASCII encoding as long as no special characters are used? Besides: The potential overhead of this function is compile-time only as it's a static string. So I don't really see a reason to start differentiating when we might get away with not using Unicode...

I'll change it though, if you insist :)

Reply:
> But isn't Unicode identical to ASCII encoding as long as no special characters are used?

Yes.

> Besides: The potential overhead of this function is compile-time only as it's a static string. So I don't really see a reason to start differentiating when we might get away with not using Unicode...
> 
> I'll change it though, if you insist :)

I agree, but we are currently using `fromLatin1()` everywhere, when we're not dealing with Unicode strings.

Reply:
Done